src/test/java/com/wikia/webdriver/testcases/adstests/TestUrlBuilder.java

```diff
@@ -1,378 +1,420 @@
+          "http://bindingofisaac.wikia.com/de/wiki/Items",
+          "http://bindingofisaac.preview.wikia.com/de/wiki/Items",
+          "http://bindingofisaac.sandbox.wikia.com/de/wiki/Items",
+          "http://bindingofisaac.sandbox-mercurydev.wikia.com/de/wiki/Items",
+          "http://bindingofisaac.dmytror.wikia-dev.pl/de/wiki/Items",
+          "http://bindingofisaac.wikia-staging.com/de/wiki/Items",
+          "de"
+      },
+      {
+          "videospielehub", "wiki/Videospiele_Hub", false, true,
+          "http://videospielehub.wikia.com/de/wiki/Videospiele_Hub",
+          "http://videospielehub.preview.wikia.com/de/wiki/Videospiele_Hub",
+          "http://videospielehub.sandbox.wikia.com/de/wiki/Videospiele_Hub",
+          "http://videospielehub.sandbox-mercurydev.wikia.com/de/wiki/Videospiele_Hub",
+          "http://videospielehub.dmytror.wikia-dev.pl/de/wiki/Videospiele_Hub",
+          "http://videospielehub.wikia-staging.com/de/wiki/Videospiele_Hub",
+          "de"
+      },
+      {
+          "jedipedia", "wiki/Jedipedia:Hauptseite", false, true,
+          "http://jedipedia.wikia.com/de/wiki/Jedipedia:Hauptseite",
+          "http://jedipedia.preview.wikia.com/de/wiki/Jedipedia:Hauptseite",
+          "http://jedipedia.sandbox.wikia.com/de/wiki/Jedipedia:Hauptseite",
+          "http://jedipedia.sandbox-mercurydev.wikia.com/de/wiki/Jedipedia:Hauptseite",
+          "http://jedipedia.dmytror.wikia-dev.pl/de/wiki/Jedipedia:Hauptseite",
+          "http://jedipedia.wikia-staging.com/de/wiki/Jedipedia:Hauptseite",
+          "de"
+      },
+      {
+          "memory-alpha", "wiki/Portal:Main", false, true,
+          "http://memory-alpha.wikia.com/wiki/Portal:Main",
+          "http://memory-alpha.preview.wikia.com/wiki/Portal:Main",
+          "http://memory-alpha.sandbox.wikia.com/wiki/Portal:Main",
+          "http://memory-alpha.sandbox-mercurydev.wikia.com/wiki/Portal:Main",
+          "http://memory-alpha.dmytror.wikia-dev.pl/wiki/Portal:Main",
+          "http://memory-alpha.wikia-staging.com/wiki/Portal:Main",
+          "en"
+      },
+      {
+          "memory-alpha", "wiki/Hauptseite", false, true,
+          "http://memory-alpha.wikia.com/de/wiki/Hauptseite",
+          "http://memory-alpha.preview.wikia.com/de/wiki/Hauptseite",
+          "http://memory-alpha.sandbox.wikia.com/de/wiki/Hauptseite",
+          "http://memory-alpha.sandbox-mercurydev.wikia.com/de/wiki/Hauptseite",
+          "http://memory-alpha.dmytror.wikia-dev.pl/de/wiki/Hauptseite",
+          "http://memory-alpha.wikia-staging.com/de/wiki/Hauptseite",
+          "de"
+      },
+  };
 
-    @Test(groups = {"TestUrlBuilder", "unitTests"})
-    public void urlBuilderTests() throws Exception{
-        Configuration.setTestValue("qs", "");
-        Constructor<UrlBuilder> c = getUrlBuilderConstructor();
+  @Test(groups = {"TestUrlBuilder", "unitTests"})
+  public void urlBuilderTests() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00112)